tests/unittest/_torch/modules/mamba/test_gdn_kernel_optimizations.py

```
@@ -0,0 +1,164 @@
+@skip_no_cuda
+@pytest.mark.parametrize("seq_len", [1, 32, 128, 1024])
+@pytest.mark.parametrize(
+    "q_dim,k_dim,v_dim,num_q_heads,head_k_dim,num_v_heads,head_v_dim",
+    [
+        (256, 256, 16, 16, 16, 16, 1),   # Qwen3.5-35B like
+        (512, 512, 64, 16, 32, 16, 4),
+    ],
+)
+@pytest.mark.parametrize("dtype", [torch.bfloat16])
+def test_split_qkv_contiguous(seq_len, q_dim, k_dim, v_dim,
+                               num_q_heads, head_k_dim, num_v_heads, head_v_dim, dtype):
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)